.drone.yml

```diff
@@ -1,33 +1,20 @@
-build:
-  image: golang:1.5
-  environment:
-    - CGO_ENABLED=0
-  commands:
-    - make deps
-    - make vet
-    - make build
-    - make test
+workspace:
+  base: /go
 
-publish:
-  coverage:
-    when:
-      branch: master
-  docker:
-    username: $$DOCKER_USER
-    password: $$DOCKER_PASS
-    email: $$DOCKER_EMAIL
-    repo: plugins/drone-elastic-beanstalk
-    tag: latest
-    when:
-      branch: master
-  docker:
-    username: $$DOCKER_USER
-    password: $$DOCKER_PASS
-    email: $$DOCKER_EMAIL
-    repo: plugins/drone-elastic-beanstalk
-    tag: develop
-    when:
-      branch: develop
+pipeline:
+    build:
+      image: golang:1.6
+      commands:
+        - CGO_ENABLED=0 go build
+    docker:
+      username: $$DOCKER_USER
```

Comment:
The secrets system changed a lot, take a look at https://github.com/drone/drone-plugin-starter/blob/master/.drone.yml